src/App.tsx

```diff
@@ -103,51 +145,54 @@ const App = () => {
       }
   }, [useDashboardTheme]);
 
+
   return (
-    <>
-      <GlobalStyle />
-      <Wrapper>
-        {!connectedProvider && (
-          <SignIn
-            onWeb3ProviderSet={async (web3Provider) => {
-              if (!web3Provider) {
-                setConnectedProvider(null);
-                return;
-              }
-
-              const web3 = new Web3(web3Provider as any);
-              // @ts-ignore
-              setConnectedProvider(web3.currentProvider)
-            }}
-            onWeb3AuthInstanceSet={setWeb3AuthInstance}
-          />
-        )}
-        {connectedProvider && (
-          <div>
-            <ToggleThemeButton onClick={() => setUseDashboardTheme(!useDashboardTheme)}>Toggle theme</ToggleThemeButton>
-            <Etherspot
-              defaultTransactionBlocks={[{ type: TRANSACTION_BLOCK_TYPE.ASSET_BRIDGE }]}
-              provider={connectedProvider}
-              chainId={chainId}
-              themeOverride={themeOverride}
-              onLogout={async () => {
-                if (!web3AuthInstance) return;
-
-                try {
-                  await web3AuthInstance.logout({ cleanup: true });
-                  web3AuthInstance.clearCache();
-                } catch (e) {
-                  //
+    <WagmiConfig client={client}>
+      <>
+        <GlobalStyle />
+        <Wrapper>
+          {!connectedProvider && (
+            <SignIn
+              onWeb3ProviderSet={async (web3Provider, isWagmi) => {
+                if (!web3Provider) {
+                  setConnectedProvider(null);
+                  return;
                 }
 
-                setConnectedProvider(null);
+                const web3 = new Web3(web3Provider as any);
+                // @ts-ignore
```

Comment:
I assume this is cos the provider types don't match, but maybe leave a comment here as to why we're ignoring ts here for future reference